prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -89,6 +89,17 @@ public async Task SendSiteRegistrationSubmissionAsync(int siteId, int businessLi
             await _emailDocumentService.SaveSiteRegistrationReview(siteId, siteRegReviewPdf);
         }
 
+        public async Task SendHealthAuthoritySiteRegistrationSubmissionDetailsToHIBCAsync(int healthAuthoritySiteId)
+        {
+
+            var email = await _emailRenderingService.RenderSiteRegistrationSubmissionEmailAsync(new LinkedEmailViewModel(null), CareSettingType.HealthAuthority);
+            email.Attachments = await _emailDocumentService.GenerateHealthAuthoritySiteRegistrationSubmissionAttachmentsAsync(healthAuthoritySiteId);
+            await Send(email);
+
+            var siteRegReviewPdf = email.Attachments.Single(a => a.Filename == "HealthAuthoritySiteRegistrationReview.pdf");
```

Comment:
since we only have one attachment, wrapping it in an array and then having to do this to get it back feels bad. It would be better to generate the one attachment and just do `email.Attachments = new [] {attachment}`

Reply:
Was this comment supposed to go to line 96 instead?

Reply:
fixed